include/swift/AST/Decl.h

```diff
@@ -554,13 +554,7 @@ class alignas(1 << DeclAlignInBits) Decl {
     RawForeignKind : 2,
 
     /// \see ClassDecl::getEmittedMembers()
-    HasForcedEmittedMembers : 1,
-
-    /// Information about the class's ancestry.
-    Ancestry : 7,
```

Comment:
Since these are used rather frequently, I would imagine making this request ExternallyCached would still be profitable. Or maybe not...